src/components/DragAndDropImage.tsx

```
@@ -0,0 +1,78 @@
+const DragAndDropImage: React.FC<DragAndDropImageProps> = ({
+  onImageUpload,
+}) => {
+  const [preview, setPreview] = useState<string | null>(null)
+  const [isDragging, setIsDragging] = useState(false)
+
+  const handleDrop = (event: React.DragEvent<HTMLDivElement>) => {
+    event.preventDefault()
+    setIsDragging(false)
+
+    const file = event.dataTransfer.files[0]
+    if (file && file.type.startsWith('image/')) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)